html/pfappserver/root/radiuslog/search.tt

```diff
@@ -8,7 +8,7 @@
       <!--th[% " class=\"$class\"" IF class %]><a href="[% c.uri_for(search_action,page_num,per_page,params)  %]">[% title | html %][% IF by == column %] <i class="icon-sort-[% IF new_direction == 'desc' %]down[% ELSE %]up[% END %]"></i>[% END %]</a></th-->
       <th[% " class=\"$class\"" IF class %]>[% title | html %][% IF by == column %] <i class="icon-sort-[% IF new_direction == 'desc' %]down[% ELSE %]up[% END %]"></i>[% END %]</th>
 [% END -%]
-      <table id="items" class="table table-condensed table-striped items">
+      <table id="RadiusAuditLog" class="table table-condensed table-striped items">
```

Comment:
HTML ids start lowercase in our code convention from what I know.

Change this to radiusAuditLog here and in radiusauditlog.js#14